src/rgw/rgw_user.cc

```diff
@@ -2299,7 +2299,7 @@ int RGWUserAdminOp_User::info(RGWRados *store, RGWUserAdminOpState& op_state,
     return ret;
 
   if (!op_state.has_existing_user())
-    return -ENOENT;
+    return -ERR_NO_SUCH_USER;
```

Comment:
I think it's not necessary to check subuser existence when operating on RGWUser. 

Maybe we should return NoSuchUser for subuser/keys/caps operations too?